admin-dev/themes/new-theme/js/pages/product/services/combinations-service.ts

```diff
@@ -24,14 +24,25 @@
  */
 
 import Router from '@components/router';
+import ServiceType from '@PSTypes/services';
```

Comment:
I just noticed, shouldn't we use `PaginationServiceType` here? I know it works because it's returned as default But maybe we can replace this one later?

@zuk3975 is actually changing this service by splitting it into two because it forces injecting a productId for every services even those that don't need it (see this commit https://github.com/PrestaShop/PrestaShop/pull/27216/commits/be9a980215a03e06c665f5d1dde383e3c6943ac6)

I guess we can clean this in julius' PR once yours will be merged